### PR TITLE
fix validator package version

### PR DESCRIPTION
**Pull request by ywrsusan**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -62,11 +62,11 @@
     "flatten-json==0.1.6",
     "boto>=2.48.0, <2.49",
     "python-magic>=0.4.13, <0.4.14",
+    "validators<=0.14.2",
```
Comment (dchhabda) [inline setup.py:65]: any reason to pick 0.14.2? I think by 0.16.0 release date, it should be good. https://libraries.io/pypi/validators. Also there is + can be removed.

Comment (ywrsusan) [reply setup.py:65]: because versions between 0.14.2 and 0.17.0 are yanked? https://pypi.org/project/validators/#history

Comment (dchhabda) [reply setup.py:65]: make sense. + sign can be removed.

Comment (ywrsusan) [reply setup.py:65]: where is the `+` sign?

Comment (dchhabda) [reply setup.py:65]: ignore, my bad

**Pull request merged**